Formula/retdec.rb

```diff
@@ -0,0 +1,64 @@
+class Retdec < Formula
+  desc "Retargetable machine-code decompiler based on LLVM"
+  homepage "https://github.com/avast/retdec"
+  url "https://github.com/avast/retdec/archive/refs/tags/v4.0.tar.gz"
+  sha256 "b26c2f71556dc4919714899eccdf82d2fefa5e0b3bc0125af664ec60ddc87023"
+  license all_of: ["MIT", "Zlib"]
+  head "https://github.com/avast/retdec.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "libtool" => :build
+  depends_on "openssl@1.1" => :build
+  depends_on "python@3.10"
+
+  on_macos do
+    depends_on xcode: :build
+    depends_on macos: :mojave
+  end
+
+  on_linux do
+    depends_on "gcc@7" => :build
+  end
+
+  def install
+    inreplace "cmake/options.cmake", "set_if_at_least_one_set(RETDEC_ENABLE_OPENSLL
+		RETDEC_ENABLE_CRYPTO)", ""
+    inreplace "deps/CMakeLists.txt", "cond_add_subdirectory(openssl RETDEC_ENABLE_OPENSLL)", ""
+    inreplace "src/crypto/CMakeLists.txt", "ALIAS crypto)", "ALIAS crypto)
+
+    find_package(OpenSSL 1.1.1 REQUIRED)"
+    inreplace "src/crypto/CMakeLists.txt", "retdec::deps::openssl-crypto", "OpenSSL::Crypto"
+    inreplace "src/crypto/retdec-crypto-config.cmake", "openssl-crypto", ""
```

Comment:
Why is this needed?

Reply:
Fix for finding OpenSSL from system so that it doesn't build it locally

Reply:
I found a fix for Mac M1: Update OpenSSL used in RetDec to 1.1.1i which supports Mac M1 (Using inreplace). RetDec 4.0 currently fetches 1.1.1c which doesn't support Mac M1. After this fix, RetDec will build OpenSSL locally and I can remove openssl dependency. Is that better than what I already have?

Reply:
It won't be merged depending on a vendored version of openssl, see https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae

If there is no acceptable release yet we'll have to wait for such a release.

Reply:
So my fix passed all checks. If vendored version is not acceptable then my previous fix for using openssl locally works then or no?

Reply:
I mean it finds the homebrew installed openssl 1.1.1 so its not vendored anymore right since its using latest homebrew openssl?